### ISSUE TITLE
HTTPS Leaking like crazy

### ISSUE THREAD

**Issue by jacobmmp** [EasyEngine/easyengine#698]
The issue is the way you implementet https with letsencrypt, you forgot to ad a default_server for the https, so right now if you only got one https site every vhost you got on the server called true https will end up pointing on the one site that got https enabled.

This is actually inflicting harm on SEO since they popup on google search results, please fix this azap, this is a major concerne to be honest.

**Comment by w33zy** [comment]
Are you trying that if you only have 1 enabled to use HTTPS and you then visit the another site on the server that does not use HTTPS, it redirects to the HTTPS site?

Am I understanding you correctly?

**Comment by jacobmmp** [comment]
yes thats what im saying :) and that is a huge flaw in the configuration of Nginx.

**Comment by oviliz** [comment]
To me that's a normal behaviour of any server.

Sent by mobile

**Comment by jacobmmp** [comment]
The normal behaviour would be it to show the Nginx Page thats show as default when you call ex the ip or a domain thats pointing on the server not having a vhost.

**Comment by oviliz** [comment]
The normal behaviour depends by the server settings really. If there is no
vhost, the server would not even respond to either HTTP or HTTPS request.

**Comment by jacobmmp** [comment]
Still the standart config should make sure HTTPS for the server got a default so you dont get leaky https all over the place.
